difference_of_squares.rb

```diff
@@ -0,0 +1,34 @@
+class Squares
+  @@num = 0
+
+  private
```

Comment:
why everything is  private? Can you please push test cases related to this exercise? I want to see test cases.